src/lib.rs

```diff
@@ -163,11 +173,16 @@ impl<'rx, 'tx> Eth<'rx, 'tx> {
                 .pm()
                 .set_bit()
         });
+
         // Flow Control Register
         self.eth_mac.macfcr.modify(|_, w| {
             // Pause time
-            w.pt().bits(0x100)
+            #[allow(unused_unsafe)]
+            unsafe {
+                w.pt().bits(0x100)
+            }
```

Comment:
is `allow(unused_unsafe)` necessary ? is unsafe necessary?

Reply:
It probably was before, but it's not needed any more.